flocker/node/agents/test/test_blockdevice.py

```diff
@@ -4119,13 +4125,14 @@ class FakeProfiledLoopbackBlockDeviceIProfiledBlockDeviceTests(
     """
 
 
-def umount(device_file):
+def umount(unmount_target):
     """
     Unmount a filesystem.
 
-    :param FilePath device_file: The device file that is mounted.
+    :param FilePath unmount_target: The device file that is mounted or
```

Comment:
Given that we're doing a second round of review though, let me know if you strongly feel like we should only document the mountpoint API and I can change it.